File: src/main/java/skywolf46/extrautility/events/interaction/PlayerLeftClickEvent.java
package skywolf46.extrautility.events.interaction;

import org.bukkit.entity.Player;
import org.bukkit.event.HandlerList;
import org.bukkit.inventory.ItemStack;
import skywolf46.extrautility.events.abstraction.AbstractPlayerItemEvent;

public class PlayerLeftClickEvent extends AbstractPlayerItemEvent {
    private static HandlerList handlerList = new HandlerList();


    public PlayerLeftClickEvent(Player who) {
        super(who);
    }

    @Override
    public HandlerList getHandlers() {
        return handlerList;
    }

    public static HandlerList getHandlerList() {
        return handlerList;
    }

}

File: src/main/kotlin/skywolf46/extrautility/events/interaction/PlayerLeftClickEvent.kt
package skywolf46.extrautility.events.interaction

import org.bukkit.entity.Player
import org.bukkit.event.HandlerList
import org.bukkit.event.player.PlayerInteractEvent
import skywolf46.extrautility.events.abstraction.AbstractPlayerItemEvent


class PlayerLeftClickEvent(ev: PlayerInteractEvent, who: Player?, val isBlockInteraction: Boolean) :
    AbstractPlayerItemEvent(ev, who) {
    companion object {
        val handlerList = HandlerList()
    }

    override fun getHandlers(): HandlerList = handlerList
}
